### PR TITLE
Adding Personality Recognition System using NLP

### PR DESCRIPTION
**Pull request by Shashankjain12**

### CODE REVIEW

File: Adrian-Popescu/README.md [hunk 1]
```diff
@@ -31,4 +31,4 @@ To install syft with pip, you can issue `pip install syft`.
 
 | Name | Slack |
 | ------ | ------ |
-| Adrian Popescu | Adrian Popescu |
\ No newline at end of file
+| Adrian Popescu | Adrian Popescu |
```
Comment (Shanmugapriya03) [inline Adrian-Popescu/README.md:34]: You have changed another person's file can you please revert this file changes!

**Review by Shanmugapriya03 — CHANGES_REQUESTED** [review]

**Review by Sajjadmanal — APPROVED** [review]

File: Adrian-Popescu/README.md (hunk 1 above)
Comment (Shanmugapriya03) [reply Adrian-Popescu/README.md:34]: Also, the folder name should be your slack name and not the name of the project

Comment (Shashankjain12) [reply Adrian-Popescu/README.md:34]: @Shanmugapriya03 this is the group project and it is mentioned to specify group project with the name of the project

Comment (Shanmugapriya03) [reply Adrian-Popescu/README.md:34]: okay I just saw that now sorry for the inconvenience but can you revert the change you made to `Adrian-Popescu/README.md`

**Pull request closed without merging**